src/Services/Database.ts

```diff
@@ -88,16 +95,8 @@ class Database {
     }
 
     private initModels(): void {
-        Example.init({
-            help: {
-                type: DataTypes.STRING(128),
-                allowNull: true,
-            },
-        } as ModelAttributes,
-        {
-            sequelize: this.sequelize,
-            tableName: "Examples",
-        } as InitOptions);
+        Example.initialize(this.sequelize);
```

Comment:
You have removed migration for example but here initializing the model. This would break :)